### ISSUE TITLE
Posted suggestions/missed tweets for English filter on Reddit

### ISSUE THREAD

**Issue by hallomatt** [plorry/VegAssist#45]
Hey there,

I found a few tweets over the weekend that didn't get caught by the English filter.
Since more people have a Reddit account than a Github account, I figured it might be a good idea to collect missed tweets / raw data over at Reddit. This way, every VegAssist follower can help improve the bot.

https://www.reddit.com/r/VegAssist/

What do you guys think?

**Comment by squeek502** [comment]
Since technical things like missed tweets or suggestions is mostly relevant to developers, using Github issues would probably be better (speaking for myself at least). Much easier for me to miss specific submissions/comments on Reddit (for example, without this issue, I would probably have never seen your thread on Reddit, even though I'm subscribed to /r/VegAssist).

It's definitely good to have a Reddit thread to collect input from non-developers, but I think it'd be best if we mirror all relevant things to Github issues. That's [what I do for other projects of mine](https://github.com/squeek502/VeganOption/issues/61).

**Comment by hallomatt** [comment]
Good point! Gonna mirror every /r/VegAssist submission I see to Github, so that a developer can look into it.

> It's definitely good to have a Reddit thread to collect input from non-developers

Yeah, this way we will catch many more missed tweets, especially in non-english languages.
How about we add this info to the project doc, so that more VegAssist followers can submit missed tweets on Reddit?

**Comment by hallomatt** [comment]
How do you guys handle the next steps? Anything I can help you with?
